### ISSUE TITLE
Barrel/Conveyances Tab Does Not Display Properly after Creating Work Order

### ISSUE THREAD

**Issue by cstevenson1** [leeensminger/DelDOT-NPDES-Field-Tool#200]
I created a pipe conveyance, then proceeded to create a barrel conveyance containing that pipe. I then created a work order for the one barrel and everything appeared to be normal. I saved the work order and the barrel conveyance then navigated to another feature so I could attempt to reopen the barrel conveyance. Using the selections tab (advanced query builder) I searched for the barrel conveyance and was able to reopen it but when I clicked on the Barrels/Conveyances tab, the normal Barrels/Conveyances form and the Work Order form appeared to be overlaying one another as seen in the snip below. 

![image](https://cloud.githubusercontent.com/assets/16919898/17697854/511269c6-6385-11e6-99ac-38f6b6c0c7c5.png)

**Comment by cstevenson1** [comment]
Melissa experienced the same issue. However, we discovered that if you hit Cancel as seen in snip 1 until you return to the main Barrels/Conveyances page as seen in snip 2, the two forms do not overlay one another. We had to restart the field tool to discover this though because after the two forms overlaid one another that initial time, they continued to do so for all barrel conveyances added afterwards making the Barrels/Conveyances tab unusable until we relaunched the field tool.

Snip 1
![image](https://cloud.githubusercontent.com/assets/16919898/17703210/e9810a3c-639e-11e6-82d4-49a7109d35ab.png)

Snip 2
![image](https://cloud.githubusercontent.com/assets/16919898/17703233/f780041c-639e-11e6-96fe-658c7adaf90d.png)
